db/db_impl.cc

```diff
@@ -4759,18 +4759,18 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
       if (!cfd->mem()->IsEmpty()) {
         // We only consider active mem table, hoping immutable memtable is
         // already in the process of flushing.
-        size_t cfd_size = cfd->mem()->ApproximateMemoryUsage();
-        if (largest_cfd == nullptr || cfd_size > largest_cfd_size) {
-          largest_cfd = cfd;
-          largest_cfd_size = cfd_size;
+        uint64_t seq = cfd->mem()->GetLatestSeqWhenCreate();
```

Comment:
Re empty memtables: of course I didn't think that this code would flush empty memtables; my problem is with the order of flushing nonempty memtables.